apps/demos/Demos/Charts/PointImage/description.md

```diff
@@ -1,2 +1,4 @@
-This demo shows that the Chart can display custom images instead of&nbsp;standard point symbols.
-<!--split-->
\ No newline at end of file
+DevExtreme Chart can display images at series points. Specify the **point**.**image** object in [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/point/image/) or [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/image/) to display point images.
```

Comment:
DevExtreme Chart can display custom images for series points. Specify the **point**.**image** object in [common series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/point/image/) or [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/image/) settings to display these images.